packages/git-encrypted/src/crypto.ts

```diff
@@ -2,7 +2,11 @@ import { arrayToHex, hexToArray } from 'enc-utils';
 import { hash, randomBytes, secretbox } from 'tweetnacl';
 import { decodeUTF8, encodeBase64, encodeUTF8 } from 'tweetnacl-util';
 import { NONCE_LENGTH } from './constants';
-import { Keys } from './types';
+import { Keys, GitBaseParamsEncrypted } from './types';
+import { doesDirectoryExist } from './utils';
+import { deriveKeys } from './keyDerivation';
+import { saveKeysToDisk } from './api/saveKeysToDisk';
+import { ensureMetaExists } from './encryptedMeta';
```

Comment:
nit: VSCode has an "organize imports" feature which sorts the imports somehow, it's neat to use, although hard to enforce.